lib/valkyrie/persistence/fedora/ordered_list.rb

```diff
@@ -68,6 +70,10 @@ def to_graph
 
       attr_reader :node_cache
 
+      # Append a node to a linked list
+      # @param source [ListNode] the node being appended
+      # @param append_node [ListNode] the node already in the existing list
+      # @return [TrueClass] whether or not the OrderedList has changed
```

Comment:
I don't think this is true. It always returns true, but I don't think the return value matters.

Reply:
Can you resolve this?